sonar-batch/src/test/java/org/sonar/batch/index/SourceDataFactoryTest.java

```diff
@@ -364,6 +364,46 @@ public void applySymbolReferences() throws Exception {
   }
 
   @Test
+  public void applySymbolReferences2() throws Exception {
```

Comment:
would it be possible to have more meaningfull name for this method and the next ?